tests/Unit/DataStructures/Test_Identity.cpp

```diff
@@ -0,0 +1,35 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <array>
+#include <catch.hpp>
+
+#include "DataStructures/Tensor/Identity.hpp"
+#include "Utilities/MakeArray.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+
+namespace {
+template <size_t Dim>
+void test_identity() {
+  const std::array<double, Dim> used_for_size = make_array<Dim>(0.0);
```

Comment:
There's no point constructing the array since you only use the 0th component of it.